sql/moz-fx-data-shared-prod/fenix_derived/firefox_android_clients_v1/checks.sql

```diff
@@ -1,10 +1,10 @@
-#fail
+#warn
 {{ is_unique(columns=["client_id"]) }}
 
-#fail
+#warn
 {{ not_null(columns=["client_id", "sample_id"], where="submission_date = @submission_date") }}
 
-#fail
+#warn
```

Comment:
This sounds like it really shoud fail the downstream?
I see how non unique client_ids or something especially with regards to this being a problem might not warrant to stop downstream, but if there is no data ... there is no data to run?

Reply:
Let's just call it "over-compensating" the other way. We will still see failures and can easily re-run the pipeline from this point.

Reply:
But I thought the whole idea of the fails was that we don't want to rerun expensive queries all the time?

Reply:
ok, row count checks are back to being a fail. Others we can assume they're okay to not pass and instead should only create a warning not stop ETL execution.